### PR TITLE
Bug 2032 - Test pull request

### PR DESCRIPTION
**Pull request by RLSWE**
Fix the bug with feature x where layout would be ambiguous in small screens

The bug was caused by not setting paddings on the bottom

### CODE REVIEW

File: db/queries/categoriesdb.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const getCategories = () => {
     });
 };
 
-const getItemsInCategory = (categoryId) => {
+const getItemsByCategory = (categoryId) => {
```
Comment (RLSWE) [inline db/queries/categoriesdb.js:14]: This name can be misleading. Let's change this to `getItemsOnCategory`

File: db/queries/categoriesdb.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const getCategories = () => {
...
   return db
     .query(`SELECT items.*, item_images.img_url FROM items
   JOIN (SELECT DISTINCT ON (item_id) id, item_id, img_url FROM item_images ORDER BY item_id, id)
```
Comment (RLSWE) [inline db/queries/categoriesdb.js:17]: Oh if we're touching this feature already, let's re-write this query

File: routes/conditions.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ const conditionsdb = require("../db/queries/conditionsdb");
 // GET /conditions   - Gets all conditions
 erouter.get("/", (req, res) => {
   conditionsdb.getConditions().then((conditions) => {
+    // Some comment that explains the code
```
Comment (RLSWE) [inline routes/conditions.js]: This comment is unnecessary as this code is obvious

**Review by RLSWE — COMMENTED** [review]
Code looks good, thanks for the contribution, left some comments

**Review by RLSWE — COMMENTED** [review]
Code looks good, thanks for the contribution, left some comments

**Review by RLSWE — COMMENTED** [review]
Code looks good, thanks for the contribution, left some comments

**Review by RLSWE — COMMENTED** [review]
Code looks good, thanks for the contribution, left some comments

**Review by RLSWE — COMMENTED** [review]
Code looks good, thanks for the contribution, left some comments

File: db/queries/categoriesdb.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ const getCategories = () => {
     });
 };
```
Comment (JennyCarroll) [inline db/queries/categoriesdb.js:13]: not good, too bad for you!

File: routes/categories.js [hunk 5]
```diff
@@ -11,7 +11,8 @@ erouter.get("/", (req, res) => {
 // GET /categories/:categoryId   - Gets all items for category
 erouter.get("/:categoryId", (req, res) => {
```
Comment (JennyCarroll) [inline routes/categories.js:12]: worst ive ever seen

**Review by JennyCarroll — CHANGES_REQUESTED** [review]

**Comment by RLSWE** [comment]
You know what, this is unnecessary

**Pull request closed without merging**